### ISSUE TITLE
Update 15 June 23

### ISSUE THREAD

**Issue by HFeenstra** [cpolhout/loqed_custom_component#14]
Installed the latest update. My lock did not come up. Removed completely integration and entity, rebooted, reinstalled integration and again can install the lock but no connection.

Hope to hear if it’s just me or others??

**Comment by Frank2787** [comment]
I have the same issue: 
![image](https://github.com/cpolhout/loqed_custom_component/assets/103364638/a5c47312-3ca6-4ba9-af6f-c89073b616cf)

Also removed the integration; reinstalled and again no connection.

**Comment by fierland** [comment]
me too

**Comment by HFeenstra** [comment]
He all,
just an update: I found the follow up integration:
Mikewoudenberg/homeassistant-Loqed 

https://github.com/mikewoudenberg/homeassistant-loqed/issues/8#issuecomment-1594132958

The above comment about a simple mistake I made. But there he also explains that his integration is a follow up on this integration. If you remove this integration and change to the above one, all will work just fine. And fewer mistakes.
